desktopAssistant.py

```diff
@@ -54,7 +54,10 @@ def assistant(command):
             url = url + 'r/' + subreddit
         webbrowser.open(url)
         print('Done!')
-
+    
+    elif "lock my PC" in command:
+        os.system("rundll32.exe user32.dll,LockWorkStation")
```

Comment:
I use a macOs, and most people use a flavour of nix systems.
I'll merge this if you handle the os differences.